pyccel/ast/type_annotations.py

```diff
@@ -348,3 +334,18 @@ def __eq__(self, o):
                     self.order == o.order
         else:
             return False
+
+#==============================================================================
+
+typenames_to_dtypes = { 'float'   : PythonNativeFloat(),
+                        'double'  : PythonNativeFloat(),
+                        'real'    : PythonNativeFloat(),
```

Comment:
Do we still want to support `'real'`?

Reply:
Probably not. But I don't think we want to get rid of it in this PR

Reply:
https://github.com/pyccel/pyccel/issues/1786